main.go

```diff
@@ -141,15 +141,16 @@ func main() {
 	}
 
 	if *gpg != "" {
-		homeDir, err := os.UserHomeDir()
+		configDir, err := os.UserConfigDir()
 		if err != nil {
-			log.Fatal("failed to find user home dir")
+			log.Fatal("failed to find user config dir")
 		}
 		basePath := *gpgConfigBasepath
 		// fallback to default location if not specified
 		if basePath == "" {
-			basePath = filepath.Join(homeDir, "AppData", "Roaming", "gnupg")
+			basePath = filepath.Join(configDir, "gnupg")
```

Comment:
According to https://pkg.go.dev/os#UserConfigDir `os.UserConfigDir()` just resolves to %appdata% on Windows,

so shouldn't this be `basePath = filepath.Join(configDir, "Local", "gnupg")`?
(or `Roaming`, depending on the gpg version, see https://github.com/BlackReloaded/wsl2-ssh-pageant/issues/33#issuecomment-1005743099)